structuralcodes/materials/concrete/_concreteMC2010.py

```diff
@@ -167,3 +173,21 @@ def Gf(self, value: float):
             value (float): the value of Gf in N/m
         """
         self._Gf = abs(value)
+
+    @property
+    def gamma_c(self) -> float:
+        """The partial factor for concrete."""
+        return self._gamma_c or 1.5
+
+    def fcd(self, alpha_cc: float) -> float:
```

Comment:
should alpha_cc have a default value of 1.0? (see 7.2.3.1.4)

Reply:
Fixed in b5f0e82